tools/docker/fabric-all-in-one/Dockerfile

```diff
@@ -34,6 +34,14 @@ RUN apk add --no-cache curl
 # The file binary is used to inspect exectubles when debugging container image issues
 RUN apk add --no-cache file
 
+# Download and setup path variables for Go
```

Comment:
Recommend to also answer the `why` (IIRC it was because you need to compile the peer binary for Fabric 2.x due to lack of CLI container or something related to that right?)

Reply:
Incorporated the changes. Please have a re-look.